src/test/java/io/confluent/examples/streams/ValidateStateWithInteractiveQueriesLambdaIntegrationTest.java

```diff
@@ -102,21 +103,21 @@ public void shouldComputeMaxValuePerKey() throws Exception {
     KStream<String, Long> input = builder.stream(inputTopic);
 
     // rolling MAX() aggregation
+    String maxStore = "max-store";
```

Comment:
nit: `final`

Reply:
nothing is final in the rest of the class and i'm not changing everything!